src/invidious/channels/community.cr

```diff
@@ -240,8 +271,10 @@ def extract_channel_community(items, *, ucid, locale, format, thin_mode)
           end
         end
       end
-      if cont = items.dig?(-1, "continuationItemRenderer", "continuationEndpoint", "continuationCommand", "token")
-        json.field "continuation", extract_channel_community_cursor(cont.as_s)
+      if !is_single_post
+        if cont = items.dig?(-1, "continuationItemRenderer", "continuationEndpoint", "continuationCommand", "token")
+          json.field "continuation", extract_channel_community_cursor(cont.as_s)
+        end
```

Comment:
Do single posts even have a `continuationItemRenderer`?

Reply:
It seems to exist but i dont think it returns anything